ietf-optical-impairment-topology.yang

```diff
@@ -399,42 +399,71 @@ module ietf-optical-impairment-topology {
           a specification in a separate equipment catalog";
       }
       container operational {
-        description "amplifier operationnal parameters";
-        leaf actual-gain {
-          type decimal64 {
-            fraction-digits 2;
+        description "amplifier operational parameters";
+        list amplifier-element {
+          description
+            "The list of parallel amplifier elements within an
+            amplifier used to amplify different frequency ranges.";
+          leaf name {
+            type string;
+            description
+              "The name of the amplifier element as specified in
+              the vendor's specification associated with the
+              type-variety.";
           }
-          units dB ;
-      mandatory true ;
-          description "..";
-        }
-        leaf tilt-target {
-          type decimal64 {
-            fraction-digits 2;
+          container frequency-range {
+            description
+              "The frequency range amplified by the amplifier
+              element.";
+            leaf lower-frequency {
+              type l0-types-ext:frequency-thz;
```

Comment:
I think that the lower and upper frequencies should be mandatory

Reply:
That would be ok for me.

The implication of this change is that the frequency range would be mandatory also in case a single amplifier is used, which was not the case in the previous version of the model.

Is this ok for everybody?

Reply:
Hi Esther, based on Italo's comment, can we consider the PR approved and open an issue regarding your request , to discuss with the others if they are fine with the proposal ?

Reply:
OK!